src/aat/resources/features/F-103/F-103_Test_Case_Access_Base_Steve.td.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_guid_": "F-103_Test_Case_Access_Base_Steve",
+	"method": "PUT",
+	"uri": "/cases/{cid}/users/{uid}",
```

Comment:
there should be an _extends_ line here to reuse content of the common, base test data object.

Reply:
Done